server/sonar-server/src/test/java/org/sonar/server/computation/step/LoadQualityProfilesStepTest.java

```diff
@@ -39,7 +39,7 @@
 import static org.sonar.db.rule.RuleTesting.XOO_X1;
 import static org.sonar.db.rule.RuleTesting.XOO_X2;
 
-public class FeedActiveRulesStepTest {
+public class LoadQualityProfilesStepTest {
 
   @Rule
   public TemporaryFolder temp = new TemporaryFolder();
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "TemporaryFolder". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS2924)